lib/stdlib/src/erl_parse.yrl

```diff
@@ -401,6 +401,20 @@ if_clauses -> if_clause ';' if_clauses : ['$1' | '$3'].
 if_clause -> guard clause_body :
 	{clause,first_anno(hd(hd('$1'))),[],'$1','$2'}.
 
+begin_expr -> 'begin' maybe_exprs 'end' : {block,?anno('$1'),'$2'}.
+begin_expr -> 'begin' maybe_exprs 'else' else_clauses 'end' : {block,?anno($1),'$2','$3'}.
```

Comment:
```suggestion
begin_expr -> 'begin' exprs 'end' : {block,?anno('$1'),'$2'}.
begin_expr -> 'begin' maybe_exprs 'end' : {block,?anno('$1'),'$2',[]}.
begin_expr -> 'begin' maybe_exprs 'else' else_clauses 'end' : {block,?anno($1),'$2','$3'}.
```
This might be cleaner?

Reply:
Does this even build?  Problem is that the lookahead won't work to distinguish between a sequence of exprs and maybe exprs.  So there should be a conflict reported between the clause in 404 and 405.  It it builds it will error out at runtime when the parser gets there because its no longer a LALR(1) grammar